components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityConfigParser.java

```diff
@@ -204,6 +212,25 @@ private void buildConfiguration() {
         }
     }
 
+    private void buildCookiesToInvalidateConfig() {
+
+        OMElement cookiesToInvalidate = this.getConfigElement(IdentityConstants.COOKIES_TO_INVALIDATE_CONFIG);
+        Iterator<OMElement> cookies = cookiesToInvalidate.getChildrenWithName(
+                new QName(IdentityCoreConstants.IDENTITY_DEFAULT_NAMESPACE, IdentityConstants.COOKIE));
+        if (cookies != null) {
+            while (cookies.hasNext()) {
+                OMElement cookie = cookies.next();
+                String cookieName = cookie.getAttributeValue(new QName(IdentityConstants.COOKIE_NAME));
+
+                if (StringUtils.isBlank(cookieName)) {
+                    throw IdentityRuntimeException.error("Cookie name not defined correctly");
```

Comment:
Invalid Configuration.  Cookie name is not defined correctly"

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3373/commits/2f9dc5c391cb14eb4538bda7de2b38da9c3c17c5